jetstream/concepts/consumers.md

```diff
@@ -2,33 +2,115 @@
-| OptStartTime | When first consuming messages from the Stream start with messages on or after this time |
-| RateLimit | The rate of message delivery in bits per second |
-| MaxAckPending | The maximum number of messages without acknowledgement that can be outstanding, once this limit is reached message delivery will be suspended |
+### AckPolicy
+How messages should be acknowledged. The server will consider an ack ony if it comes within the Ack Wait window.
+If the ack is not received in time, the message(s) will be redelivered.
 
+#### AckExplicit
+
+This is the default policy. It means that each individual message must be acknowledged. It is the only allowed option for pull consumers.
+
+#### AckNone
+
+You do not have to ack any messages, the server will assume ack on delivery.
+
+#### AckAll
+
+If you receive a series of messages, you only have to ack the last one you received.
+All the previous messages received are automatically acknowledged.
+
+### AckWait
+
+Ack Wait is the time in nanoseconds that the server will wait for an ack for any individual message.
+If an ack is not received in time, the message will be redelivered.
+
+
+### DeliverPolicy / OptStartSeq / OptStartTime
+
+When a consumer is first created, it can specify where in the stream it wants to start receiving messages.
+This is the `DeliverPolicy` and it's options are as follows:
+
+#### DeliverAll
+
+All is the default policy. The consumer will start receiving from the earliest available message.
+
+#### DeliverLast
+
+The consumer will start receiving messages with the last message added to the stream, so the very last message in the stream when the server realizes the consumer is ready.
+
+#### DeliverNew
+
+The consumer will only start receiving messages that were created after the consumer was created.
+
+#### DeliverByStartSequence
+
+When first consuming messages from the Stream, start at this particular message in the set. The consumer is required to specify `OptStartSeq`, the sequence number to start on. It will receive the closest available sequence if that message was removed based on the stream limit policy.
+
+#### DeliverByStartTime
+
+When first consuming messages from the Stream start with messages on or after this time. The consumer is required to specify `OptStartTime`, the time in the stream to start at. It will receive the closest available message on or after that time.
+
+### DeliverSubject
+
+The subject to deliver observed messages. Useful to set up an alternate subject for a regular NatsSubcriber can listen on that subject. Not allowed for pull subscriptions.
+Deliver subject essentially creates an alias core NATS subject for the stream. This means a core NATS subscriber could be set up to receive messages on the DeliverSubject,
+starting wherever the DeliverPolicy was configured for the consumer. You could use a core NATS subscriber to access the stream by its original subject, but that would always start at
```

Comment:
no you really should never set up a core nats subscriber on the original subject of the stream, you should only do that using the js context. Don't even mention this because it will not go well

Reply:
yeah, this whole part is removed